timemachine/cpp/src/kernels/k_logsumexp.cu

```diff
@@ -21,4 +21,31 @@ void __global__ k_exp_sub_max(
 template void __global__ k_exp_sub_max<float>(const int, const float *, const float *, float *);
 template void __global__ k_exp_sub_max<double>(const int, const double *, const double *, double *);
```

Comment:
Is `k_exp_sub_max` still used anywhere?

Reply:
Nope, removed

Reply:
Good catch, removed in https://github.com/proteneer/timemachine/pull/1239/commits/77d48810c1b4f270e961ce2870e7093286865abd